src/getLocale.ts

```diff
@@ -1,17 +1,16 @@
+import { getCookie, setSessionCookie } from './cookies';
 import { isString } from './isString';
 import { storage } from './storage';
 import type { CountryCode } from './types/countries';
 
-const KEY = 'gu.geolocation';
+const KEY = 'GU_geo_country';
+const KEY_OVERRIDE = 'gu.geo.override';
 const URL = 'https://api.nextgen.guardianapps.co.uk/geolocation';
 
 // best guess that we have a valid code, without actually shipping the entire list
 const isValidCountryCode = (country: unknown) =>
 	isString(country) && /^[A-Z]{2}$/.test(country as string);
```

Comment:
i know it's not part of this PR but it would be good to cache this regex rather than recreating it every time this is called

Reply:
which one is the expensive part? should we memoize the results of calling `test`, or do you mean just the pattern bit?

Reply:
yeah that's one, saves recreating a regex for `/^[A-Z]{2}$/` each time it runs – thanks :)